source/sass/buyers-guide/views/catalog.scss

```diff
@@ -217,12 +217,7 @@ body {
 
     @media (min-width: $breakpoint) {
       display: flex;
-      height: var(--sticky-height);
       align-items: center;
-
-      &.creep-o-meter-moved {
-        height: var(--sticky-height);
-      }
```

Comment:
@fessehaye Sorry, I don't understand? Maybe my question was not clear enough. I was wondering why we are removing these styles? Because of the class names this appears unrelated to me.

Reply:
![image](https://user-images.githubusercontent.com/2374073/200905633-fa0b1e96-140e-45c3-aa4d-670ea1daa522.png)

It creates a huge amount of space between the category header and the row included the privacy ding since the creep-face use to take up a row on previous iteration of PNI

Reply:
Thanks for the explanation @fessehaye. Yes, I had seen that somewhere. Thanks for fixing this.